doc/python/marker-style.md

```diff
@@ -378,6 +378,31 @@ fig.show()
 
 ```
 
+#### Open Marker Colors
+
+In the previous example, each marker has two colors, a marker color (set in Plotly Express with `color="species"`)  and a line color (set on the line with `color="DarkSlateGrey"`. When using open markers, like `"diamond-open"` in the following example, the marker always uses only the marker color. Adding a line color won't update the marker. 
```

Comment:
```suggestion
In the previous example, each marker has two colors, a marker color (set in Plotly Express with `color="species"`)  and a line color (set on the line with `color="DarkSlateGrey"`. All open markers, like "diamond-open" in the following example, have a transparent fill, which means you can specify only one color.  Specify this color using the marker color parameter. This controls the the outline color and any dot or cross. For open markers, the line color does nothing.
```
Hi @rl-utility-man, looks good. Thanks! Just updated it slightly to speak to the user

Reply:
Other than the "the the" the text here looks great 😜